src/components/ConnectWallet/ConnectWallet.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, ButtonProps } from '@chakra-ui/button';
+import React from 'react';
+import { useWeb3 } from '../../hooks';
```

Comment:
I'd vote on putting the path aliases in haha!

Reply:
Great suggestion!

Reply:
We are going to divide the hooks and components into separate packages through a monorepo so that'd probably solve that.